RPIPs/RPIP-36.md

```diff
@@ -0,0 +1,94 @@
+|---------------------------------------|------------------------------------------------------------------------|------------------|----------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Incentives Management Committee (IMC) | To support low slippage exchange between ETH and rETH                  | 2022-08-25       | [RPIP-20](RPIP-20), [RPIP-10](RPIP-10) | [Mainnet](https://app.safe.global/home?safe=eth:0xb867EA3bBC909954d737019FEf5AB25dFDb38CB9), [Arbitrum](https://app.safe.global/home?safe=arb1:0xd7102A3744c302f167c53621453516345bC460d7) | [Tracking Sheet](https://docs.google.com/spreadsheets/d/1jLGkoiSjNr1UgG1spAH9ppBfrk_FqjJXg9CIw35nLCA/edit#gid=1325210165)                                                                                                                                                                |
+| Grants Management Committee (GMC)     | To distribute funding to further the goals of the Rocket Pool protocol | 2022-10-11       | [RPIP-26](RPIP-26), [RPIP-10](RPIP-10) | [Mainnet](https://app.safe.global/home?safe=eth:0x6efD08303F42EDb68F2D6464BCdCA0824e1C813a)                                                                                                | [Tracking Sheet](https://docs.google.com/spreadsheets/d/1oxOioCXoVIqY4aFS9mEgHMgK18QLEfJcwlZk2-l6cfc/edit?usp=sharing), [Standard Operating Procedures](https://docs.google.com/document/d/1YNG0nhC04jDonO42hT_MZ0sQT-3EIU04HK9Fwrkzs30/edit#heading=h.s7ggzumy4f02), [Records](RPIP-29) |
+
+## Incentives Management Committee
+
+### Current Membership
+
+| Member Since | Members          | Member Type | Comment                                                                                                                                                                              | Address                                                                            |
+|--------------|------------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
+|   2023-09-22 | DrWorm           | Voting      | Selected as [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af) runner-up after Langers steps down. | [Address](https://etherscan.io/address/0xC701b5283822f84a2B358f79a4526FD6E1b8C395) |
```

Comment:
Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.

Reply:
Agree